lib/tenstorrent/bh_arc/smbus_target.c

```diff
@@ -157,61 +160,98 @@ static SmbusData smbus_data = {
 	.state = kSmbusStateIdle,
 };
 
+int32_t UpdateArcStateHandler(const uint8_t *data, uint8_t size)
+{
+	const uint8_t sig0 = 0xDE;
+	const uint8_t sig1 = 0xAF;
```

Comment:
Where do these magic numbers come from? A comment would be useful. Or if they are derived somehow, it would be good to know how they are derived. Are they part of some spec?